docs/sphinx/specification/cudaq/operations.rst

```diff
@@ -99,4 +99,6 @@ qubit state is a :code:`|0>` (positive polarity) or :code:`|1>` (negative polari
 By default all control qubits are assumed to convey positive polarity. 
 The syntax for negating the polarity is the not operator preceeding the
 control qubit (e.g., :code:`x<cudaq::ctrl>(!q, r)`, 
-for :code:`cudaq::qubits` :code:`q` and :code:`r`).
\ No newline at end of file
+for :code:`cudaq::qubits` :code:`q` and :code:`r`). Negating the polarity of
+control qubits is supported in :code:`swap` and the gates in sets :code:`NAME`
```

Comment:
by this we mean that you are able to negate any control qubits on a swap operation, but you are not able to negate any of the 2 target qubits. Might be good to add a bit more detail.